build/simp-environment.spec

```diff
@@ -42,11 +42,11 @@ Buildroot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
 Requires: libselinux-utils
 Requires: openssl
 Requires: policycoreutils
-Requires: simp-rsync >= 6.2.0-0
-Requires: simp-utils >= 6.0.0-0
 Requires(pre,preun,post,postun): simp-adapter
+Requires(pre,post): puppet
 Requires(post): coreutils
 Requires(post): createrepo
+Requires(post): facter
```

Comment:
I don't believe there is still a facter rpm

Reply:
that doesn't mean facter must be in a rpm (same with puppet).  It means the puppet-agent rpm advertises that it provides facter and puppet.  (rpm -q --provides puppet-agent to see what it provides)

Reply:
Makes sense.